variation-commons-mongodb/src/test/java/uk/ac/ebi/eva/commons/mongodb/repositories/VariantRepositoryTest.java

```diff
@@ -138,8 +138,8 @@ public void testNonExistentVariantIdIsNotFound() {
     @Test
     public void testVariantRegionIsFound() {
         String chr = "11";
-        int start = 190013;
-        int end = 190013;
+        long start = 190013;
+        long end = 190013;
```

Comment:
The region declarations is not uniform: some are one-liners and others span through multiple lines. Given how simple this case is I would suggest making them all one-liners.

Reply:
Inlined all chr, start and end declarations, but not this, because the chr, start and end variables are being used also in the asserts